src/components/WwwFrame/TheHeader.vue

```diff
@@ -572,18 +589,29 @@ export default {
 		},
 	},
 	apollo: {
-		query: headerQuery,
-		preFetch: true,
-		result({ data }) {
-			this.isVisitor = !data?.my?.userAccount?.id;
-			this.isBorrower = data?.my?.isBorrower ?? false;
-			this.loanId = data?.my?.mostRecentBorrowedLoan?.id ?? null;
-			this.trusteeId = data?.my?.trustee?.id ?? null;
-			this.basketCount = data?.shop?.nonTrivialItemCount ?? 0;
-			this.balance = Math.floor(data?.my?.userAccount?.balance ?? 0);
-			this.profilePic = data?.my?.lender?.image?.url ?? '';
-			this.profilePicId = data?.my?.lender?.image?.id ?? null;
-			this.basketState = data || {};
+		preFetch(config, client, { cookieStore }) {
+			return client.query({
+				query: headerQuery,
+			}).then(({ data }) => {
+				this.isVisitor = !data?.my?.userAccount?.id;
```

Comment:
`this` is not available in the prefetch function (it's run outside of the component context before the component is created). So we still need the `result` function and the `query: headerQuery` line. The rest of this `then` function will still work.

Reply:
done!